lib/opensusebasetest.pm

```diff
@@ -652,6 +650,32 @@ sub wait_grub {
     if (get_var('ONLINE_MIGRATION')) {
         push @tags, 'migration-source-system-grub2';
     }
+    # if there was a request to enroll or remove a certificate used for SecureBoot process
+    if (get_var('_EXPECT_MOK') && get_var('UEFI')) {
```

Comment:
I wouldn't expect `_EXPECT_MOK` to be set on non-UEFI

Reply:
I would be surprised if it had happened. I just thought it might raise awareness that this is strictly related to `UEFI` as this part of the code is in library. Not only that, but I am not that sure if everyone knows what MOK is. Ofc, I have no issue to simplify the condition.

Reply:
> Not only that, but I am not that sure if everyone knows what MOK is. Ofc, I have no issue to simplify the condition.

You could call the variable `_EXPECT_EFI_MOK_MANAGER`

Reply:
Done!